src/security/engines/BrowserKeyStore.js

```diff
@@ -121,101 +150,73 @@ class BrowserKeyStore {
         if (!BROWSER) {
             throw new Error("Browser Only");
         }
-        const member = this._loadMember(memberId);
-        if (!member) {
-            if (!member) {
-                throw new Error(`member ${memberId} not found`);
-            }
-        }
-        BrowserKeyStore.setActiveMemberId(memberId);
-        var list = [];
-        for (var level in member) {
-            if (member.hasOwnProperty(level)) {
-                list.push(member[level]);
-            }
-        }
-        return list;
+        const store = await BrowserKeyStore._getObjectStore(MEMBER_KEY_STORE);
```

Comment:
consider using helper method to get member id from store + return in promise to reduce code repetition